docs-src/concepts.md

```diff
@@ -112,42 +114,123 @@ TODO
 
 ### GatewayClass
 
-`GatewayClass` is cluster-scoped resource defined by the infrastructure ops.
-This resource represents a category of Gateways that can be instantiated.
+`GatewayClass` ([source code][gatewayclass-src]) is cluster-scoped resource
+defined by the infrastructure ops. This resource represents a class of Gateways
```

Comment:
/infrastructure ops/infrastructure provider/ to be consistent with persona naming.